### DISCUSSION TITLE
New Command idea

### DISCUSSION THREAD

**Discussion by HimadriChakra12** [ajeetdsouza/zoxide#948] [Ideas]
Zoxide should get a new command where it list all dir and sub dir which will let us select where to go
![Untitled](https://github.com/user-attachments/assets/ca87d5bb-7506-4404-a027-7115617ec7e3)

**Comment by ajeetdsouza** [comment]
Isn't this what the `zi` command provided by zoxide does?

**Reply by HimadriChakra12** [reply]
Nope.
'Zi' provides a history of the directory I have cd-ed to.
The following shows the directories you can cd to within the directory you are in.

**Reply by ajeetdsouza** [reply]
This is already provided by fzf:

```bash
alias zx='cd "$(fzf --walker=dir)"'
```

**Reply by HimadriChakra12** [reply]
I Didn't knew that! Its good. But This Command has a weak point which is sorting directories (at least in powershell even with sorting commands, Idk about bash/zsh).
